BraveRewardsUI/Tipping/TippingSelectionViewController.swift

```diff
@@ -0,0 +1,65 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import UIKit
+import BraveRewards
+
+class TippingSelectionViewController: OptionsSelectionViewController {
+  
+  private let state: RewardsState?
+  
+  // Hide the super constructor with `DisplayableOption`
+  // in favour of our constructor with `BatValue`
+  private override init(options: [DisplayableOption],
+                        selectedOptionIndex: Int = 0,
+                        optionSelected: @escaping (_ selectedIndex: Int) -> Void) {
+    self.state = nil
+    super.init(options: options, selectedOptionIndex: selectedOptionIndex, optionSelected: optionSelected)
+  }
+  
+  init(state: RewardsState, options: [BATValue],
+       selectedOptionIndex: Int = 0,
+       optionSelected: @escaping (_ selectedIndex: Int) -> Void) {
+    self.state = state
+    super.init(options: options, selectedOptionIndex: selectedOptionIndex, optionSelected: optionSelected)
+  }
+  
+  override func viewDidLoad() {
+    super.viewDidLoad()
+    
+    self.title = "Recurring Tips"
```

Comment:
1. Should be localized/placed in `Strings.swift`
2. Lets go with singular `Recurring Tip` instead

Reply:
Fixed.